daemon/lxd.go

```diff
@@ -166,3 +169,75 @@ func (l *LXD) DeleteContainer(hostname string) (bool, error) {
 
 	return true, nil
 }
+
+func (l *LXD) CreateContainerBootstrapFile(container pfmodel.Container, fullPath string) error {
+	var content string
+	mode := 644
+	contentType := "file"
+	writeMode := "overwrite"
+
+	for _, bs := range container.Bootstrappers {
+		content, mode = util.SetupBootstrapFileContent(bs, content, mode)
```

Comment:
I don't understand the flow on this function,
does it means that if we have more than one bootstrappers, only the last bootstrappers will be written on the container?

Reply:
Also what is the purpose of setting mode to `644` above? it seems the `SetupBootstrapFile` method override that